### DISCUSSION TITLE
AttributeError: Can't get attribute 'C3k2' on <module 'ultralytics.nn.modules.block' from '/usr/local/lib/python3.9/dist-packages/ultralytics/nn/modules/block.py'>

### DISCUSSION THREAD

**Discussion by Keven-Don** [ultralytics/ultralytics#17602] [Q&A]
guys I had met some problem which error message like that 

Traceback (most recent call last):
  File "/home/ross/yo11/src/predict.py", line 9, in <module>
    model = YOLO("/home/ross/yo11/model/yolo11x-pose.pt")
  File "/usr/local/lib/python3.9/dist-packages/ultralytics/models/yolo/model.py", line 23, in __init__
    super().__init__(model=model, task=task, verbose=verbose)
  File "/usr/local/lib/python3.9/dist-packages/ultralytics/engine/model.py", line 152, in __init__
    self._load(model, task=task)
  File "/usr/local/lib/python3.9/dist-packages/ultralytics/engine/model.py", line 241, in _load
    self.model, self.ckpt = attempt_load_one_weight(weights)
  File "/usr/local/lib/python3.9/dist-packages/ultralytics/nn/tasks.py", line 806, in attempt_load_one_weight
    ckpt, weight = torch_safe_load(weight)  # load ckpt
  File "/usr/local/lib/python3.9/dist-packages/ultralytics/nn/tasks.py", line 732, in torch_safe_load
    ckpt = torch.load(file, map_location="cpu")
  File "/usr/local/lib/python3.9/dist-packages/torch/serialization.py", line 789, in load
    return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)
  File "/usr/local/lib/python3.9/dist-packages/torch/serialization.py", line 1131, in _load
    result = unpickler.load()
  File "/usr/local/lib/python3.9/dist-packages/torch/serialization.py", line 1124, in find_class
    return super().find_class(mod_name, name)
AttributeError: Can't get attribute 'C3k2' on <module 'ultralytics.nn.modules.block' from '/usr/local/lib/python3.9/dist-packages/ultralytics/nn/modules/block.py'>
and my environment version is : #
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main    defaults
_openmp_mutex             5.1                       1_gnu    defaults
bzip2                     1.0.8                h5eee18b_6    defaults
ca-certificates           2024.9.24            h06a4308_0    defaults
certifi                   2024.8.30                pypi_0    pypi
charset-normalizer        3.4.0                    pypi_0    pypi
contourpy                 1.3.1                    pypi_0    pypi
cycler                    0.12.1                   pypi_0    pypi
filelock                  3.16.1                   pypi_0    pypi
fonttools                 4.55.0                   pypi_0    pypi
fsspec                    2024.10.0                pypi_0    pypi
idna                      3.10                     pypi_0    pypi
jinja2                    3.1.4                    pypi_0    pypi
kiwisolver                1.4.7                    pypi_0    pypi
ld_impl_linux-64          2.40                 h12ee557_0    defaults
libffi                    3.4.4                h6a678d5_1    defaults
libgcc-ng                 11.2.0               h1234567_1    defaults
libgomp                   11.2.0               h1234567_1    defaults
libstdcxx-ng              11.2.0               h1234567_1    defaults
libuuid                   1.41.5               h5eee18b_0    defaults
markupsafe                3.0.2                    pypi_0    pypi
matplotlib                3.9.2                    pypi_0    pypi
mpmath                    1.3.0                    pypi_0    pypi
ncurses                   6.4                  h6a678d5_0    defaults
networkx                  3.4.2                    pypi_0    pypi
numpy                     2.1.3                    pypi_0    pypi
nvidia-cublas-cu12        12.4.5.8                 pypi_0    pypi
nvidia-cuda-cupti-cu12    12.4.127                 pypi_0    pypi
nvidia-cuda-nvrtc-cu12    12.4.127                 pypi_0    pypi
nvidia-cuda-runtime-cu12  12.4.127                 pypi_0    pypi
nvidia-cudnn-cu12         9.1.0.70                 pypi_0    pypi
nvidia-cufft-cu12         11.2.1.3                 pypi_0    pypi
nvidia-curand-cu12        10.3.5.147               pypi_0    pypi
nvidia-cusolver-cu12      11.6.1.9                 pypi_0    pypi
nvidia-cusparse-cu12      12.3.1.170               pypi_0    pypi
nvidia-nccl-cu12          2.21.5                   pypi_0    pypi
nvidia-nvjitlink-cu12     12.4.127                 pypi_0    pypi
nvidia-nvtx-cu12          12.4.127                 pypi_0    pypi
opencv-python             4.10.0.84                pypi_0    pypi
openssl                   3.0.15               h5eee18b_0    defaults
packaging                 24.2                     pypi_0    pypi
pandas                    2.2.3                    pypi_0    pypi
pillow                    11.0.0                   pypi_0    pypi
pip                       24.2            py310h06a4308_0    defaults
psutil                    6.1.0                    pypi_0    pypi
py-cpuinfo                9.0.0                    pypi_0    pypi
pyparsing                 3.2.0                    pypi_0    pypi
python                    3.10.15              he870216_1    defaults
python-dateutil           2.9.0.post0              pypi_0    pypi
pytz                      2024.2                   pypi_0    pypi
pyyaml                    6.0.2                    pypi_0    pypi
readline                  8.2                  h5eee18b_0    defaults
requests                  2.32.3                   pypi_0    pypi
scipy                     1.14.1                   pypi_0    pypi
seaborn                   0.13.2                   pypi_0    pypi
setuptools                75.1.0          py310h06a4308_0    defaults
six                       1.16.0                   pypi_0    pypi
sqlite                    3.45.3               h5eee18b_0    defaults
sympy                     1.13.1                   pypi_0    pypi
tk                        8.6.14               h39e8969_0    defaults
torch                     2.5.1                    pypi_0    pypi
torchvision               0.20.1                   pypi_0    pypi
tqdm                      4.67.0                   pypi_0    pypi
triton                    3.1.0                    pypi_0    pypi
typing-extensions         4.12.2                   pypi_0    pypi
tzdata                    2024.2                   pypi_0    pypi
ultralytics               8.3.32                   pypi_0    pypi
ultralytics-thop          2.0.11                   pypi_0    pypi
urllib3                   2.2.3                    pypi_0    pypi
wheel                     0.44.0          py310h06a4308_0    defaults
xz                        5.4.6                h5eee18b_1    defaults
zlib                      1.2.13               h5eee18b_1    defaults

**Comment by UltralyticsAssistant** [comment]
👋 Hello @Keven-Don, thank you for bringing this to our attention! 🚀 We're here to help.

It looks like you encountered an `AttributeError` related to the `C3k2` attribute. To assist you better, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/). This will help us understand the context of the issue you're facing more clearly.

In the meantime, ensure you have installed the latest version of Ultralytics with:

```bash
pip install -U ultralytics
```

This ensures that any recent fixes are applied and may resolve your issue. Also, please check that your environment meets all necessary requirements as detailed in our [repository's requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml).

If you haven't already, explore our [Docs](https://docs.ultralytics.com) for detailed usage examples and insights. For further community support, join us on [Discord](https://ultralytics.com/discord) 🎧 or engage in discussions on [Discourse](https://community.ultralytics.com) and [Subreddit](https://reddit.com/r/ultralytics).

🏗️ Our Ultralytics engineers will review your query soon to provide additional assistance. Appreciate your patience and support. Stay tuned!

**Comment by glenn-jocher** [comment]
@Keven-Don the error you're encountering with the `C3k2` attribute suggests a potential compatibility issue with your model weights or the Ultralytics package version. Please ensure you are using the latest version of the Ultralytics package by running `pip install ultralytics --upgrade`. Additionally, verify that your model weights are compatible with this version. If the issue persists, try re-downloading your model weights and check the [Ultralytics YOLO documentation](https://docs.ultralytics.com) for any updates or changes in model architecture.
